src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -2242,7 +2242,7 @@ void GetRefrigerationInput(EnergyPlusData &state)
             }
 
             ++AlphaNum; // A6
-            if (lAlphaBlanks(AlphaNum)) {
+            if (!lAlphaBlanks(AlphaNum)) {
```

Comment:
What's this?

Reply:
I was trying to bisect something I was seeing locally so I reverted this line. I didn't work.